tests/common/fixtures/services.py

```diff
@@ -0,0 +1,25 @@
+from unittest.mock import create_autospec
+
+import pytest
+
+from h.services.search_index import SearchIndexService
+
+# Allow people to mass import everything without getting things we don't
+# mean them to like pytest.
+__all__ = ("service_mocker", "search_index")
+
+
+@pytest.fixture
+def service_mocker(pyramid_config):
+    def service_mocker(class_, name):
```

Comment:
```suggestion
    def service_mocker(service_class, name):
```

It can be nice to avoid things like `class_` when possible

Reply:
:+1: Done